theme/less/_header.less

```diff
@@ -0,0 +1,63 @@
+#header-top
+{
+  padding: 25px;
+}
```

Comment:
Currently whole row looks off visually:
- in figma height of whole top header seem to be 75px, now it's 100+px.
- Logo and language selector are missalinged.

Reply:
The issue was from default margins of `#header-logo p`. Setting it to 0 and `align-self:center` solves the issue.